MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -255,6 +255,19 @@
                 }
 				break;
 #endif
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_DRIVER_PROPERTIES: {
+				auto* physicalDeviceDriverProps = (VkPhysicalDeviceDriverPropertiesKHR*)next;
+				char mvkVersion[16], vkVersion[16];
+				vkGetVersionStringsMVK(mvkVersion, sizeof(mvkVersion), vkVersion, sizeof(vkVersion));
+				strcpy(physicalDeviceDriverProps->driverName, "MoltenVK");
+				strcpy(physicalDeviceDriverProps->driverInfo, mvkVersion);
```

Comment:
Might as well just use `mvkGetMoltenVKVersionString(MVK_VERSION).c_str()` right in the `strcpy()` arg, and skip `mvkVersion` and `vkGetVersionStringsMVK()` altogether.